site/content/docs/main/restore-hooks.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Restore Hooks"
+layout: docs
+---
+
+Velero now supports Restore Hooks, custom actions that can be executed during or after the restore process. There are two kinds of Restore Hooks:
```

Comment:
We should take out "now" and just say "Velero supports" or "Velero 1.5 added support for..."

Also, we say in the style guide not to reference things in the future, is this something that we want to include early, or hold post on including until it out? for ExeecRestorHooks https://velero.io/docs/main/style-guide/#avoid-statements-about-the-future

Reply:
The exec restore hook is WIP and is part of the features that we designed for It is a feature that many in the community are waiting for.
We will be cutting a patch release to get that out.

Reply:
Ok, thanks for the context @ashish-amarnath. I'm fine either way, but wanted to raise it, since it technically goes against the style guide (exceptions are perfectly fine though)  :)

Reply:
I agree with this being against the guide. But I don't want folks to interpret this as RestoreHooks is fully completed.